File: InsertionSort/JS/insertionSort.js
const insertionSort = arr => {
    let n = arr.length,
            tmp;

    for (let i=1; i<n; i++){
        tmp = arr[i];
        let j=i;

        while (j>0){

            if(arr[j-1]>tmp){
                arr[j]=arr[j-1];
                j--;
            } else break;
        }
        arr[j]=tmp;
    }
    return arr;
}

File: InsertionSort/C++/insertionSort.cpp
void insertionSort(int arr[], int size) {
    int tmp, j;
    for (int i = 1; i < size; i++) {
        tmp = arr[i];
        j = i;
        while (j > 0) {
            if (arr[j - 1] > tmp) {
                arr[j] = arr[j - 1];
                j--;
            }
            else break;
        }
        arr[j] = tmp;
    }
}